src/org-manager/store/reducers/org-pending.reducer.ts

```diff
@@ -59,12 +62,21 @@ export function reducer(
       };
     }
 
+    case PendingOrgActionTypes.APPROVE_PENDING_ORGANISATIONS_FAIL: {
+      const responseMessages = action.payload;
```

Comment:
dont put all the error message object returned by the server by maybe just a message and status code